exchange.js

```diff
@@ -48,6 +56,11 @@ const getConversion = function (amount, rate) {
     const result = `ARS $ ${amount * rate}`
     const resultEl = document.querySelector('#result-outcome')
     resultEl.innerHTML = result
+
+    const timestampJSON = localStorage.getItem('timestampMessage')
+    const lastTimestamp = JSON.parse(timestampJSON)
```

Comment:
timestampJSON is a string. No need to parse it. Also, change the name from timestampJSON to timestamp since it is misleading

Reply:
done